rewrite-java/src/main/java/org/openrewrite/java/cleanup/UnnecessaryParenthesesVisitor.java

```diff
@@ -59,8 +60,16 @@ public <T extends J> J visitParentheses(J.Parentheses<T> parens, P ctx) {
 
         assert par != null;
         if (par instanceof J.Parentheses) {
-            if (getCursor().getParentTreeCursor().getValue() instanceof J.Parentheses) {
+            J parentCursorValue = getCursor().getParentTreeCursor().getValue();
+            if (parentCursorValue instanceof J.Parentheses) {
                 return ((J.Parentheses<?>) par).getTree().withPrefix(Space.EMPTY);
+            } else if (parentCursorValue instanceof J.MethodInvocation) {
+                J tree = ((J.Parentheses<?>) par).getTree();
+                Space outside = par.getPrefix();
+                Space inside = tree.getPrefix();
+                return tree.withPrefix(Space.build(
+                        outside.getWhitespace() + inside.getWhitespace(),
+                        ListUtils.concatAll(outside.getComments(), inside.getComments())));
```

Comment:
This seemed like something we ought to have a `merge` utility for in `Space`, as there could be comments or whitespace either side of the parentheses that we want to retain.